front/src/applications/stdcm/components/SimulationReportSheetV2.tsx

```diff
@@ -290,30 +294,37 @@ const SimulationReportSheetV2 = ({
                       <TD style={styles.simulation.chColumn}>{step.ch}</TD>
                     </View>
                     <View style={styles.simulation.trackWidth}>
-                      {/* <TD style={styles.simulation.td}>{step.metadata?.trackName}</TD> */}
+                      <TD style={styles.simulation.td}>{step.trackName}</TD>
                     </View>
                     <View style={styles.simulation.endWidth}>
                       <TD style={styles.simulation.stopColumn}>
-                        {isLastStep /* || step.stop !== '0' */ ? step.time : ''}
+                        {isLastStep || step.duration !== 0 ? step.time : ''}
                       </TD>
                     </View>
                     <View style={styles.simulation.passageWidth}>
-                      <TD style={styles.simulation.stopColumn}>
+                      <TD
+                        style={{
+                          ...(step.duration !== 0 && !isLastStep
+                            ? {
+                                width: `${step.duration < 600 && step.duration >= 60 ? 60 : 70}px`,
+                                ...styles.simulation.blueStop,
+                              }
+                            : styles.simulation.stopColumn),
+                        }}
+                      >
                         {
                           // eslint-disable-next-line no-nested-ternary
                           !isFirstStep && !isLastStep
-                            ? // ? step.stop !== '0'
-                              //   ? /* getStopDurationTime(step.stop) */ 'test'
-                              step.time
+                            ? step.duration !== 0
+                              ? getStopDurationTime(step.duration)
```

Comment:
You placed this util function in time manipulation but its name is specific to your usecase. Either rename it to a more generic name that can be resued in the app or move the util in utils.ts in stdcm folder.

Reply:
done